src/2XXXR0_a_motivation_scope_and_plan_for_a_physical_quantities_and_units_library.md

```diff
@@ -44,8 +44,11 @@ the best we can get today in the C++ language.
 
 ## Dominik Berner
 
-Author of [@SI_LIB].
+Dominik is a strong believer that the C++ language can provide very high safety guarantees when programming through strong typing - A type error catched during compilation saves hours of debugging. Since the last 15 years he mainly codes in C++ an actively follows its evolution through the new standards. 
```

Comment:
```suggestion
Dominik is a strong believer that the C++ language can provide very high safety guarantees when programming through strong typing; a type error catched during compilation saves hours of debugging. Since the last 15 years, he mainly codes in C++ and actively follows its evolution through the new standards. 
```